src/spartan_app.erl

```diff
@@ -141,4 +144,8 @@ parse_ipv4_addres_with_port_test() ->
     %% Fallback to default
     ?assert({{8, 8, 8, 8}, 12345} == parse_ipv4_address_with_port("8.8.8.8", 12345)).
 
--endif.
\ No newline at end of file
+parse_ipv4_address_test() ->
+    ?assert({127, 0, 0, 1} == parse_ipv4_address("127.0.0.1")),
+    ?assert([{127, 0, 0, 1}, {2, 2, 2, 2}] == lists:map(fun parse_ipv4_address/1, ["127.0.0.1", "2.2.2.2"])).
```

Comment:
It seems like `parse_ipv4_address` can take binary as well, maybe add a test case for that?